public/scripts/slash-commands.js

```diff
@@ -733,6 +733,47 @@ export function initDefaultSlashCommands() {
         ],
         helpString: 'Unhides a message from the prompt.',
     }));
+    SlashCommandParser.addCommandObject(SlashCommand.fromProps({
+        name: 'member-get',
+        aliases: ['getmember', 'memberget'],
+        callback: (async ({field}, arg) => {
+            if (!selected_group) {
+                toastr.warning('Cannot run /member-get command outside of a group chat.');
+                return '';
+            }
+            if (field === undefined) {
+                throw new Error('\'/member-get field=\' argument required!');
+            }
+            if (!['name', 'index', 'id', 'avatar'].includes(field)) {
+                throw new Error(`Invalid '/member-get field=' argument '${field}' specified!`);
+            }
+            const isId = !isNaN(parseInt(arg));
+            const groupMember = findGroupMemberId(arg, true);
+            if (!groupMember) {
+                toastr.warn(`No group member found using ${isId?'id':'string'} ${arg}`);
+                return '';
+            }
+            return groupMember[field];
+        }),
+        namedArgumentList: [
+            SlashCommandNamedArgument.fromProps({
+                name: 'field',
+                description: 'Whether to retrieve the name, index, id, or avatar.',
+                typeList: [ARGUMENT_TYPE.STRING],
+                isRequired: true,
+                enumList: ['name', 'index', 'id', 'avatar'],
```

Comment:
For clarity, I'd make actual enum value classes and explain what `index` and `id` mean. For coders it could be clear - but I think additional documentation help doesn't hurt.